src/vscode-bicep/src/commands/deploy.ts

```diff
@@ -157,18 +158,36 @@ export class DeployCommand implements Command {
         }
       }
     } catch (err) {
-      this.outputChannelManager.appendToOutputChannel(
-        err instanceof UserCancelledError
-          ? `Deployment canceled for ${documentPath}.`
-          : `Deployment failed for ${documentPath}. ${parseError(err).message}`
-      );
+      let errorMessage: string;
+
+      if (err instanceof UserCancelledError) {
```

Comment:
"instanceof UserCancelledError" can have issues in the context of webpack.  Better to use "parseError(err).IsUserCancelled"

Reply:
Updated here:
https://github.com/Azure/bicep/commit/924e414abc9c65a549ef7ebd1ede7c7dec203268

Thanks!